### ISSUE TITLE
Dependency Problem on Deploy

### ISSUE THREAD

**Issue by pikilon** [simondotm/nx-firebase#173]
### Steps to reproduce

- Nx 17 with firebase-functions App
- import `lodash/kebabCase` versio `^4.17.21`
- use it
- **Builds correctly**

### Current result
- failed to deploy

### Expected behaviour
- deploys without problems

### Observations
- removing the import make deployment work again
- I think is related with the generated [package.json dependency here](https://cloud.google.com/functions/docs/writing/specifying-dependencies-nodejs)

**Comment by simondotm** [comment]
Yeah I think I've had this before, I believe the solution is to switch to `lodash-es` https://www.npmjs.com/package/lodash-es

This whole esm thing is a bit of a pain.

**Comment by simondotm** [comment]
Closing this.
